### PR TITLE
euclid benchmark

### PR DESCRIPTION
**Pull request by Pat-Lafon**

### CODE REVIEW

File: docs/tools/bench.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ The current benchmarks are:
 * `sum-sq-diff`: Output the difference between the sum of the squares of the first *n* natural numbers and the square of their sum.
 * `sqrt`: Implements the [Newton-Raphson Method][newton] of approximating the square root of a number to arbitrary precision
 * `mat-mul`: Multiplies two `nxn` matrices using the [naive][matmul] matrix multiplication algorithm. The matrices are randomly generated using a [linear congruential generator][rng].
+* 'euclid': Calculates the greatest common divisor between two large numbers using the [Euclidean Algorithm][euclid] with a helper function for the modulo operator.
```
Comment (sampsyo) [inline docs/tools/bench.md:22]: Try using backticks (Markdown code syntax).

**Review by sampsyo — COMMENTED** [review]
Looking good so far! The CI is complaining about the output, however—can you try running Turnt locally to make sure the output is up to date?

**Comment by Pat-Lafon** [comment]
You must have newlines at the end of your .out/.prof files!

**Review by sampsyo — APPROVED** [review]
Excellent!

**Pull request merged**